components/shotAccordion.js

```diff
@@ -133,6 +144,9 @@ function ShotAccordion(props) {
                 {props.shot[props.drillInfo.requirements[0].name]}{" "}
                 {props.drillInfo.requirements[0].distanceMeasure}
               </Text>
+              {props.drillInfo.requirements[0].type === "inputtedPutt" && (
```

Comment:
use distanceMeasure here? like just add distanceMeasure to target on the database. It seems wrong tho...